CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ Important Changes
 
 .. _`Move to Red Hat`: https://molecule.readthedocs.io/en/latest/contributing.html#move-to-red-hat
 .. _`quay.io`: https://quay.io/repository/ansible/molecule
+.. Molecule can now be called as a python module (python -m molecule)
```

Comment:
Hey, it's a syntax for external hyperlinks in RST, meaning that you put a label + link like this and then use it to highlight some link in the text.

Ref: http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets

Reply:
@ssbarnea I've fixed this myself